.github/workflows/build.yml

```diff
@@ -25,6 +25,11 @@ jobs:
         with:
           dotnet-version: '6.0.x'
 
+      - name: Install .NET SDK 7.0.x
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '7.0.x'
+
```

Comment:
```suggestion
        with:
          dotnet-version: |
            '6.0.x'
            '7.0.x'
```

Reply:
adjusted, needs to be without quotes for multiple versions.